### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by rpspringuel**
Mostly I'm just pulling what were Gna resources over here so that we're not linking to the old site (and thus elevating it's search ranking).

### CODE REVIEW

File: installation-mac.html [hunk 1]
```diff
@@ -73,7 +73,8 @@
 <li>under Preferences, click on Downloads and install the Command Line Tools</li>
 <li>download and install <a href="http://www.macports.org/">MacPorts</a> (free)</li>
 <li>once installed, in the TERMINAL, do <code>sudo port selfupdate</code> to get the latest sources packages</li>
-<li>do <code>sudo port install libtool autoconf automake flex bison libxml2</code>.</li>
+<li>do <code>sudo port install libtool autoconf automake flex bison</code>.</li>
+<li>For gregorio versions &lt;4.0, is you are using <span style="font-family: monospace;">--enable-xml-read</span> then this additional dependency is needed: <code>sudo port install libxml2</code>.</li>
```
Comment (henryso) [inline installation-mac.html]: I would actually say "prior to 4.0" rather than "<" here.

File: installation-mac.html [hunk 2]
```diff
@@ -84,6 +85,8 @@
 
 <p>You have a choice between versions of Gregorio. To obtain the sources of the stable version, you just need to download the sources from the <a href="https://github.com/gregorio-project/gregorio/releases">download page</a>, and decompress the archive.</p>
 
+<p><b>Nota bene:</b> There are three source code files on the download page.  Two have the name "Source Code" (once in zip and one in tar.gz format).  These files are images of the repository at the time of the release which are added automatically by GitHub.  As a result, the process for building from these files is identical to the process for building from a clone of the repository itself.  The other source code file, "gregorio-x.x.x.tar.bz2", is a special distribution version of the source which has several files, which do not depend on your particular system's architecture, pre-built for you.  As a result, it requires fewer tools to build.</p>
```
Comment (henryso) [inline installation-mac.html]: I'm no wordsmith so I'm a bit at a loss as to _how_, but I think this should somehow stress that you typically want the gregorio-x.x.x.tar.bz2 file.  Maybe mention it first so it's the first thing the reader sees?

Comment (henryso) [inline installation-mac.html]: Note that this line also fixes #5.

**Comment by rpspringuel** [comment]
Made some changes to address the comments.

**Comment by rpspringuel** [comment]
Fixed.

**Comment by henryso** [comment]
Looks good to me.

**Pull request merged**